content/en/deployment/lambda/index.md

```diff
@@ -0,0 +1,326 @@
+---
+title: AWS Lambda Deployment Overview
+linktitle: AWS Lambda Overview
+weight: 1
+description: >
+ Learn what an Armory CD-as-a-Service deployment to AWS Lambda is, how CD-as-a-Service connects to your AWS Account, and how deployment works.
+---
+
+
+## What a deployment is
+
+A _deployment_ encompasses the manifests, artifacts, configuration, and actions that deliver your code to remote environments. You can configure a deployment to deliver your [AWS Lambda](https://docs.aws.amazon.com/lambda) function to a single environment or multiple environments, either in sequence or in parallel depending on your [deployment configuration]({{<ref "deployment/create-deploy-config" >}}).
+
+You define your CD-as-a-Service deployment configuration in a YAML file, which you store within your source control, enabling code-like management. You trigger deployments using the Armory CLI, either from your CI system or your workstation. Although CD-as-a-Service requires a separate deployment configuration file for each app, you can deploy multiple Kubernetes Deployment objects together as part of a single app. 
+
+## How deployment works
+
+{{< figure src="lambda-deploy.webp" width=80%" height="80%" >}}
+
+CD-as-a-Service starts a deployment with a target environment, which is a combination of AWS account and region. such as development, that does not depend on another environment. Then deployment progresses through the steps, conditions, and environments defined in your deployment process. 
+
+CD-as-a-Service automatically rolls back when:
+
```

Comment:
I think we should cover HOW CD-as-a-Service does rollbacks here (https://armory.atlassian.net/browse/CDAAS-1971). Specifically:
1) if the user has specified a lambda alias to use for routing traffic, the alias is pointed back to the old version
2) if the user has not specified a lambda alias to use for routing traffic, a new version is published with the old configuration, so that the 'latest' version of the lambda function has the same configuration as before the deployment started.